Source/DafnyDriver/Legacy/DafnyBackwardsCompatibleCli.cs

```diff
@@ -78,12 +78,11 @@ private static ILegacyParseArguments TryLegacyArgumentParser(
     if (!keywordForNewMode.Contains(first)) {
       if (first.Length > 0 && first[0] != '/' && first[0] != '-' && !File.Exists(first) &&
           first.IndexOf('.') == -1) {
-        dafnyOptions.OutputWriter.WriteLine(
-          "*** Error: '{0}': The first input must be a command or a legacy option or file with supported extension",
-          first);
+        await dafnyOptions.OutputWriter.Status(
```

Comment:
nit: Should this be a `.Exception(...)` or `.Error(...)`? (And same for the `ProverException` below.)

Reply:
`Exception` is meant for Dafny bugs, and `Error` is currently for forcing things to go to `stderr`. I think we could change what goes to stdout and stderr but I'll leave that for later.